backend/src/tasks/intrigue-ident.ts

```diff
@@ -7,37 +7,36 @@ import { spawnSync } from 'child_process';
 
 const intrigueIdent = async (domain: Domain): Promise<void> => {
   console.log('Domain', domain.name);
-  for (const service of domain.services) {
-    if (service.port !== 80 && service.port !== 443) {
-      continue;
-    }
-    const url =
-      service.port === 443 ? `https://${domain.name}` : `http://${domain.name}`;
-    const { stdout, stderr, status } = spawnSync(
-      'intrigue-ident',
-      ['--uri', url, '--json'],
-      {
-        env: {
-          ...process.env,
-          HTTP_PROXY: process.env.GLOBAL_AGENT_HTTP_PROXY,
-          HTTPS_PROXY: process.env.GLOBAL_AGENT_HTTP_PROXY
-        }
+  const ports = domain.services.map((service) => service.port);
+  let service: Service;
+  if (ports.includes(443))
+    service = domain.services.find((service) => service.port === 443)!;
+  else service = domain.services.find((service) => service.port === 80)!;
```

Comment:
This means that we'll scan port 80 by default, even if there are no services available. Is that ok?

Reply:
From getLiveWebsites, we know that either 80 or 443 will exist